app/src/Components/Event/EventDetails.tsx

```diff
@@ -18,7 +18,7 @@ import EventTime from './EventTime'
 import ImageFullWidth from '../ImageFullWidth'
 import ImageSwiper from '../ImageSwiper'
 import Title from '../Title'
-import Video from '../Video'
+import Video from '../../NativeBase/Components/Video'
```

Comment:
Tiny thing but the normal way to reference this would be `@/NativeBase/Components/Video`